README.md

```diff
@@ -8,8 +8,6 @@
 A curated list of awesome Crystal code and resources. Inspired by [awesome](https://github.com/sindresorhus/awesome) and [awesome-awesomeness](https://github.com/bayandin/awesome-awesomeness).
 The goal is to have projects mostly stable and useful for the community.
 
-Search [Crystal Shards](https://crystalshards.org) or follow announcements [Crystal [ANN]](https://crystal-ann.com) for more.
-
```

Comment:
Could you please update this line rather than remove it completely:

```diff
-Search [Crystal Shards](https://crystalshards.org) or follow announcements [Crystal [ANN]](https://crystal-ann.com) for more.
+Search shards at [shards.info](https://shards.info) or follow announcements [Crystal [ANN]](https://crystal-ann.com) for more.

```